sql/validation.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+include "./db-connection.php";
+
+$logfile = fopen("logfile.txt", "w");
+
+$valid = array();
+$errors = array();
+
+define("MAX_LENGTH_SUBJECT", 128);
+define("MAX_LENGTH_LECTURER", 128);
+define("MAX_LENGTH_DESCR", 1024);
```

Comment:
I like that you have defined separate variables for maximum lengths of subject, lecturer and description, instead of using magic numbers.